client/src/components/ElementEditor/InlineEditForm.js

```diff
@@ -42,17 +43,20 @@ class InlineEditForm extends PureComponent {
   }
 
   render() {
-    const { elementId, extraClass, onClick, onFormInit, formHasState } = this.props;
+    const { element, extraClass, onClick, onFormInit, formHasState } = this.props;
     const { loadingError } = this.state;
 
     const classNames = classnames('element-editor-editform', extraClass);
-    const schemaUrl = loadElementSchemaValue('schemaUrl', elementId);
+    const schemaUrl = loadElementSchemaValue('schemaUrl', element.id);
 
     const formProps = {
       formTag: 'div',
       schemaUrl,
       identifier: 'element',
       refetchSchemaOnMount: !formHasState,
+      // Reload the form any time the element version changes, so other react components
+      // within the form have their state completely recreated
+      // refetchSchemaCriteria: JSON.stringify([element.version, element.isPublished, element.isLiveVersion]),
```

Comment:
Shouldn't this be uncommented?

Reply:
How did I... Yes it should lol. I don't remember even commenting it- will fix that.